### PR TITLE
Chunk flow batches

### PR DESCRIPTION
**Pull request by DatGuyJonathan**

### CODE REVIEW

File: apps/framework-cli/src/cli/settings.rs [hunk 1]
```diff
@@ -57,6 +58,7 @@ impl Default for Telemetry {
 pub struct Settings {
     #[serde(default)]
     pub logger: LoggerSettings,
+    #[allow(dead_code)]
     #[serde(default)]
     pub features: Features,
```
Comment (DatGuyJonathan) [inline apps/framework-cli/src/cli/settings.rs]: Do we need this for anything?

**Review by phiSgr — APPROVED** [review]

File: apps/framework-cli/src/cli/settings.rs (hunk 1 above)
Comment (callicles) [reply apps/framework-cli/src/cli/settings.rs]: Not anymore - we could reuse but it is not currently used

File: apps/framework-cli/src/cli/settings.rs [hunk 2]
```diff
@@ -23,6 +23,7 @@ const ENVIRONMENT_VARIABLE_PREFIX: &str = "MOOSE";
 
 #[derive(Deserialize, Debug)]
 pub struct Features {
+    #[allow(dead_code)]
     pub coming_soon_wall: bool,
```
Comment (callicles) [inline apps/framework-cli/src/cli/settings.rs]: This we should remove

**Review by callicles — APPROVED** [review]
Love this - will you do the same for the python version

**Comment by DatGuyJonathan** [comment]
> Love this - will you do the same for the python version

Sure, added a ticket for it.

**Pull request merged**